### DISCUSSION TITLE
How does one skin the slider end misses

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#26029] [Q&A] [answered]
I cannot figure out what to name the file

**Comment by Axilotl17** [comment]
the one found in #25951

**Reply by Axilotl17** [reply]
or does `this.ScaleTo(1, 100, Easing.In);` just scale the normal one with no skinable option

**Reply by bdach** [reply]
The above is correct. No skinnable option for now.

**Reply by peppy** [reply, marked as answer]
Now skinnable!

You can skin via `slidertickmiss@2x.png` and `sliderendmiss@2x.png`.
